### DISCUSSION TITLE
Cylindrical coordinates?

### DISCUSSION THREAD

**Discussion by nballering** [dullemond/radmc3d-2.0#25] [General]
Dear @dullemond,

I'm wondering if you have plans to implement cylindrical coordinates in the (near) future? I've encountered some situations where cylindrical coordinates would have been preferred when setting up disk models. For one, I imported the results of a disk chemical model (on a cylindrical grid) into radmc-3d, which required some careful interpolation. For another, I modeled disks that were radially narrow but vertically extended, which were awkward to sample on a spherical grid.

Thanks,
Nick Ballering

**Comment by dullemond** [comment]
Dear Nick,
no, I do not plan to implement cylindrical coordinates (I used to plan it, but I abandoned this plan). But I have now implemented unstructured grids in 3D (only in 3D), although this is in a separate branch (still beta). You could try that out, if you think it would help you. But I do not believe that cylindrical grids are better for disks than spherical ones. Yes, you may need to do some interpolations, but in Nature there are no disks that are radially narrower than their vertical extent. 
Best wishes,
Kees

**Reply by nballering** [reply]
Thanks for the info, Kees.
Best, Nick.
